python/test/unit/language/test_conversions.py

```diff
@@ -344,8 +344,6 @@ def test_typeconvert_downcast(src_dtype, dst_dtype, rounding, max_repr, device):
         if dst_dtype == 'float8e4nv':
             if not rounding == 'rtne':
                 pytest.skip("float8e4nv downcast tests only supported with RTNE rounding on AMDGPU")
-            if not is_hip_cdna4() and src_dtype == 'bfloat16':
```

Comment:
We can delete L344 - L346 too given all casting to `float8e4nv` are with `rtne` rounding mode?